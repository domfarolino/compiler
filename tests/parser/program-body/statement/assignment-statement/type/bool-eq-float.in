program dominic is

  bool target;
  float expression;

begin

  target := expression;

end program.
