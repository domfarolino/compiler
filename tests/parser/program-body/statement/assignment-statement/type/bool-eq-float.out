[1;31mLine 8: Assignment statement target (bool) is not compatible with expression (float)
Line 8: Error parsing program body
[0m